src/Bot.js

```diff
@@ -61,11 +65,11 @@ class Bot {
      * @param  {Date}    date  Date of revival, normally now.
      */
     reviveChannels(date) {
-        if(! date instanceof Date){
+        if(! (date instanceof Date)){
             throw new TypeError('Date is not a Date');
         }
         this.channels.forEach(async channel => {
-            if (date.getTime() - channel.updated.getTime() > channel.delay * 1000) {
+            if (date - channel.updated > channel.delay * 1000) {
```

Comment:
This still works?

Reply:
Yeah, `Date`s get coerced to timestamps when used with arithmetic operators. It's a little bit gross, js dynamic typing is like that, but it works as would be intuitive